tests/e2e/specs/dashboard/dashboard-noscript.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Site Kit noscript notice', () => {
+	beforeEach( async () => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-splash' );
+	} );
+
+	describe( 'When Javascript is enabled', () => {
+		it( 'Should not display noscript notice', async () => {
+			await expect( page ).not.toMatchElement( '.googlesitekit-noscript' );
+		} );
+	} );
+
+	describe( 'When Javascript is disabled', () => {
+		beforeAll( async () => {
+			await page.setJavaScriptEnabled( false );
+		} );
+
+		afterAll( async () => {
+			await page.setJavaScriptEnabled( true );
+		} );
+
+		it( 'Should not display plugin html', async () => {
+			await expect( page ).toMatchElement( '[id^=js-googlesitekit-]', { visible: false } );
```

Comment:
In addition to checking if these are visible, I'd check the contents of a selector here to ensure it's empty and doesn't contain the React app.

Reply:
There is a progress bar

https://github.com/google/site-kit-wp/pull/1150/files#diff-b6de1dbc42d4247d06044aa0ce40895bR286

Reply:
Fixed. Check lines below.